### ISSUE TITLE
pokemon games in general

### ISSUE THREAD

**Issue by EnzoLMauro** [Gericom/GBARunner2#318]
i tried to play all pokemon games but none work, they just get the gba startup screen and the screen goes white and freezes

**Comment by LucasGPP** [comment]
i tried too with a dsi and n3ds, but it seems not to work. also, it looks like the devs have abandoned it

**Comment by CRsquared64** [comment]
works fine on DSlite using an R4 for me

**Comment by HenryNordin** [comment]
All Pokemon games work for me (DSi XL with Twilight menu).

**Comment by LucasGPP** [comment]
> All Pokemon games work for me (DSi XL with Twilight menu).

What did you do? I also have a nintendo dsi xl and tried to connect with a new 3ds xl. The gba pokemon games didn't work:(

**Comment by HenryNordin** [comment]
From what I remembered I just installed twilightmenu like instructed on https://github.com/DS-Homebrew/TwilightMenu/releases and it worked as it is supposed to work. Also when you say "..and tried to connect with a new 3ds xl." do you mean that you tried to trade/battle with a different console? - But that in general it works?

**Comment by LucasGPP** [comment]
Yes, i been trying to trade/battle with mt friend. But that's the point, we can do that on ds pokemon games, but that doesn't work with the gba ones. As a matter of fact, it doesn't work with any gba games, witch makes me think that i got something wrong when installing it, but i don't know. Singleplayer works fine

**Comment by HenryNordin** [comment]
DS (even fat or lite which could natively play GBA) never had link cable support as they don't have the link-cable which the GBA uses for multiplayer - hence no multiplayer. 


However I do know that GBArunner had some link cable emulation over Wifi, but It's very experimental from my memory so not something I recommend.

**Comment by LucasGPP** [comment]
Do you know any other thing that can emulate that? And also, sorry it has been a while since i looked after all these things, so i have forgotten most of it. 
I recall seen this https://gbatemp.net/threads/gbarunner2.451970/page-94#post-8768597
Can you take look at that?
Maybe i did something wrong
